lesson-3/script.js

```diff
@@ -0,0 +1,111 @@
+let dateUser;
+let dateUserIsNumber;
+let userAge;
+
+function usersBirthYear() {
+
+  function validateUserBirthDate() {
+    return (
+      !dateUser || dateUser < 1900 || dateUser > 2005 || isNaN(dateUserIsNumber)
+    );
+  }
+  
+  do {
+    dateUser = prompt("Enter please your birth date");
+    dateUserIsNumber = Number(dateUser);
+    
+    if (validateUserBirthDate(dateUser)) {
+      alert("It is a pity that you did not want to enter your birth date...");
+    }
+  } while (validateUserBirthDate(dateUser));
+  
+
+  userAge = new Date().getFullYear() - dateUser;
+};
+
+function validateUserCitySport(validate) {
+  if (!validate || validate == " ") {
+    return true;
+  }
+};
+
+let userCity;
+let otherCity;
+let capitalCity;
+let toLowerCaseCity;
+
+function usersCity() {
+  do {
+    userCity = prompt("What city do you live in?");
+
+    if (validateUserCitySport(userCity)) {
+      alert("It is a pity that you did not want to enter your city...");
+    }
+  } while (validateUserCitySport(userCity));
+
+  toLowerCaseCity = userCity.toLowerCase();
+
+  switch (toLowerCaseCity) {
+    case "kiev":
+      capitalCity = "You live in the capital of Ukraine";
+      break;
+    case "kyiv":
+      capitalCity = "You live in the capital of Ukraine";
+      break;
```

Comment:
DRY (Don't repeat yourself). Look how it should be
```suggestion
    case "kiev":
    case "kyiv":
      capitalCity = "You live in the capital of Ukraine";
      break;
```